.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
```

Comment:
The `types` field under `on` is commented out. If you want to trigger the workflow only when a pull request is opened or synchronized, you should uncomment these lines.